addons/binding/org.openhab.binding.amazonechocontrol/README.md

```diff
@@ -115,11 +122,16 @@ It will be configured at runtime by using the save channel to store the current
 | amazonMusicPlayListId | String      | W         | echo, echoshow, echospot, wha | Write Only! Start playing of a Amazon Music playlist by specifying it's id od stops playing if a empty string was provided. Selection will only work in PaperUI
 | amazonMusic           | Switch      | R/W         | echo, echoshow, echospot, wha | Start playing of the last used Amazon Music song (works after at least one song was started after the openhab start)
 | remind                | String      | R/W         | echo, echoshow, echospot      | Write Only! Speak the reminder and sends a notification to the Alexa app (Currently the reminder is played and notified two times, this seems to be a bug in the amazon software)
+| nextReminder          | DateTime    | R           | echo, echoshow, echospot      | Next reminder on the device
+| playAlarmSound        | String      | W           | echo, echoshow, echospot      | Write Only! Plays ans Alarm sound
+| nextAlarm             | DateTime    | R           | echo, echoshow, echospot      | Next alarm on the device
+| nextMusicAlarm        | DateTime    | R           | echo, echoshow, echospot      | Next music alarm on the device
+| nextTimer             | DateTime    | R           | echo, echoshow, echospot      | Next timer on the device
 | startRoutine          | Switch      | W         | echo, echoshow, echospot      | Write Only! Type in what you normally say to Alexa without the preceding "Alexa," 
 | musicProviderId       | String      | R/W         | echo, echoshow, echospot      | Current Music provider
 | playMusicVoiceCommand | String      | W         | echo, echoshow, echospot      | Write Only! Voice command as text. E.g. 'Yesterday from the Beatles' 
 | startCommand          | String      | W         | echo, echoshow, echospot      | Write Only! Used to start anything. Available options: Weather, Traffic, GoodMorning, SingASong, TellStory, FlashBriefing and FlashBriefing.<FlahshbriefingDeviceID> (Note: The options are case sensitive)
-| textToSpeech          | String      | W         | echo, echoshow, echospot      | Write Only! Write some text to this channel and alexa will speak it 
+| textToSpeech          | String      | W         | echo, echoshow, echospot      | Write Only! Write some text to this channel and alexa will speak it. It is possible to use plain text or SSML: e.g. <speak>I want to tell you a secret.<amazon:effect name="whispered">I am not a real human.</amazon:effect></speak>
```

Comment:
Looks like the Markdown renderer eats your `<speak>` tags so the example should be quoted.

```suggestion
| textToSpeech          | String      | W         | echo, echoshow, echospot      | Write Only! Write some text to this channel and alexa will speak it. It is possible to use plain text or SSML: e.g. `<speak>I want to tell you a secret.<amazon:effect name="whispered">I am not a real human.</amazon:effect></speak>`
```